packages/inscription-view/src/components/widgets/tag/Tags.tsx

```diff
@@ -146,7 +146,10 @@ const Tags = (props: {
                     ref={inputRef}
                     value={addValue}
                     aria-label={t('tags.newTag')}
-                    style={{ width: inputValue.length * 8 > 28 ? inputValue.length * 8 : 28 }}
+                    style={{
+                      minWidth: t('tags.new').length + 'ch',
+                      width: inputValue.length * 8 > 28 ? inputValue.length * 8 : 28
+                    }}
```

Comment:
field-sizing: content;
  @supports not (field-sizing: content) {
    width: 100px;
  }